config/locales/i18n/es.yml

```diff
@@ -3,6 +3,6 @@ es:
   i18n:
     language: Idioma
     locale:
-      en: English
+      en: Inglés
       es: Español
-      fr: Français
+      fr: Francés
```

Comment:
These are used in the language picker so that we show "English" as an option when we are displaying the page in Spanish. This change should probably be reverted.